index.html

```diff
@@ -28,6 +26,149 @@ <h2>I'am a Software developer</h2>
+  <section class="tags">
+  <img src="image/css.png" alt="">
+  <img src="image/html.png" alt="">
+  <img src="image/bootstrap.png" alt="">
+  <img src="image/ruby.png" alt=""> <br>
+  <button>See Project</button>
+  </section>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  <section class="rectangle">
+  <h1>About me</h1>
+  <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+  If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+  <button>Get My Resume</button>
```

Comment:
- [ ] Kindly apply the correct button interactions as shown in the template so as to improve the user interface 👍 
- Button interaction:
![Annotation 2022-03-26 202828](https://user-images.githubusercontent.com/85113567/160254553-abc6ed1d-6564-48d3-91da-6e25a63a2020.jpg)